Pear/View/FeedbackView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  FeedbackView.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 3/21/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class FeedbackView: UIView {
+
+    // MARK: - Private View Vars
+    private var arrowbackgroundView = UIView()
+    private let feedbackTableView = UITableView()
+    private let feedbackBackgroundView = UIView()
+    private var arrowView: UIView!
```

Comment:
```suggestion
    private let arrowView = UIView()
```

Frame is zero by default.